src/main.js

```diff
@@ -96,16 +110,22 @@ const outputHTML = envSpecEntriesArray => {
   if (envSpecEntriesArray) {
     envSpecEntriesToPrint = envSpecEntriesArray.map(element => {
       toPrint = renderLabelForEntry(element.name);
+      //if element has default value
+      if (element.defaultVal) {
```

Comment:
Two comments here:

1. We should rename `defaultVal` to `defaultValue`
2. We should not put an `if` statement here, as it looks like we are choosing between types and default values. That's not right. As described in https://github.com/sourcelair/env-spec-js/issues/5, default values can be used in conjunction with types, so we should handle them elsewhere (I guess inside the `if` clause of typed values)